conformance/tests/conformance_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build conformance
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package tests
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/gateway-api/apis/v1alpha2"
+	"sigs.k8s.io/gateway-api/apis/v1beta1"
+	"sigs.k8s.io/gateway-api/conformance/tests"
+	"sigs.k8s.io/gateway-api/conformance/utils/flags"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+)
+
+func TestConformance(t *testing.T) {
+	g := NewGomegaWithT(t)
+	cfg, err := config.GetConfig()
+	g.Expect(err).To(BeNil())
+
+	client, err := client.New(cfg, client.Options{})
+	g.Expect(err).To(BeNil())
+
+	g.Expect(v1alpha2.AddToScheme(client.Scheme()).ToSucceed())
```

Comment:
this should be written like:
```
	g.Expect(v1alpha2.AddToScheme(client.Scheme())).To(Succeed())
	g.Expect(v1beta1.AddToScheme(client.Scheme())).To(Succeed())
```
otherwise, there is an error:
```
docker run --network=kind --rm -v ${HOME}/.kube/kind:/root/.kube conformance-test-runner:latest \
                go test -v . -tags conformance -args --gateway-class=nginx --supported-features=Gateway,HTTPRoute
# github.com/nginxinc/nginx-kubernetes-gateway/conformance/tests [github.com/nginxinc/nginx-kubernetes-gateway/conformance/tests.test]
./conformance_test.go:41:49: v1alpha2.AddToScheme(client.Scheme()).ToSucceed undefined (type error has no field or method ToSucceed)
./conformance_test.go:42:48: v1beta1.AddToScheme(client.Scheme()).ToSucceed undefined (type error has no field or method ToSucceed)
```

Reply:
great catch 👍🏼